Week3/homework/step2-4.js

```diff
@@ -1,10 +1,13 @@
 'use strict';
 
-function Dog() {
-  // add your code here
+function Dog(name, color) {
+  this.name = name;
+  this.numLegs = 4;
+  this.enemies = ['cats'];
+  this.color = color;
 }
 
-const hound = new Dog();
+const hound = new Dog('Jimmy', 'black');
```

Comment:
Okay, no offence taken 😄.

Reply:
:) it was one of names from Lyndas video lesson (or FCC challenge).